djangocms_navigation/admin.py

```diff
@@ -59,10 +71,191 @@ class MenuContentAdmin(admin.ModelAdmin):
     form = MenuContentForm
     menu_model = Menu
     menu_item_model = MenuItem
-    list_display = ["title", "get_menuitem_link", "get_preview_link"]
     list_display_links = None
     list_filter = (LanguageFilter, )
 
+    class Media:
+        css = {"all": ("djangocms_versioning/css/actions.css", "djangocms_version_locking/css/version-locking.css",)}
+
+    def get_version(self, obj):
+        """
+        Return the latest version of a given object
+        :param obj: MenuContent instance
+        :return: Latest Version linked with MenuContent instance
+        """
+        return obj.versions.all()[0]
+
+    def get_versioning_state(self, obj):
```

Comment:
get_versioning_state and get_state_display are the same thing, this is duplicated.

Reply:
This one is also mistakenly "Lock Status" which is duplicated.